WordPress/WordPressTest/BlockEditorSettingsServiceTests.swift

```diff
@@ -191,9 +231,89 @@ class BlockEditorSettingsServiceTests: XCTestCase {
         XCTAssertEqual(self.blog.blockEditorSettings?.checksum, originalChecksum)
     }
 
+    func testFetchBlockEditorSettings_OrgSite_NoPlugin() {
+        blog = BlogBuilder(context).build()
+
+        try! FeatureFlagOverrideStore().override(FeatureFlag.globalStyleSettings, withValue: true)
+        let mockedResponse = mockedData(withFilename: blockSettingsNOTThemeJSONResponseFilename)
+        let mockOrgRemoteApi = MockWordPressOrgRestApi()
+        service = BlockEditorSettingsService(blog: blog, remoteAPI: mockOrgRemoteApi, context: context)
+
+        let waitExpectation = expectation(description: "Theme should be successfully fetched")
+        service.fetchSettings { _ in
+            waitExpectation.fulfill()
+        }
+
+        // The app will call the none-experimental path first but fail because of compatibility reasons
+        mockOrgRemoteApi.completionPassedIn!(.failure(NSError(domain: "test", code: 404, userInfo: nil)), HTTPURLResponse())
+        // The app will then retry the experimental path.
```

Comment:
The test looks great. We just need to update the comment to reflect the removal of the experimental path.

Reply:
Good callout I missed this one.